docs/pages/blog/hello-grid-version-2.md

```diff
@@ -0,0 +1,144 @@
+
+### 💥 Breaking change
+
+The negative margin in `Grid v2` spreads equally on all sides by default (same as the `Grid` in Material UI v4).
+
+:::info
+We believe that the migration from Grid `v1` to `v2` will be smooth for most of the cases.
+:::
+
+### 🚀 New features
+
+#### [Preventing scrollbar](/material-ui/react-grid2/#disable-the-scrollbar)
+
+A new prop called `disableEqualOverflow` is added to properly prevent the scrollbar issue in small viewports.
+
+#### [Offset](/material-ui/react-grid2/#offset)
+
+We have added the long-awaited offset feature to `Grid v2`. Thanks to CSS variables, it lets us implement the feature with just a few lines of code.
+
+### ✨ Improvements
+
+#### Simplifying logic
+
+Some parts of the calculation are delegated to CSS by using CSS variables and `calc()` together. They reduces the need of javascript and makes the component scalable.
+
+For example, the `Grid v1` currently calculates the width by reading the value from a React context. However, with CSS variables in place, we can remove the React context and hand over the work to CSS instead:
+
+```js
+{
+  // --Grid-columns is defined in the grid container
+  width: `calc(100% * ${value} / var(--Grid-columns))`,
+}
+```
+
+Another example is how the padding is calculated in grid items:
+
+In `Grid v1`, grid items has no information about the padding it should have because the `spacing` prop is defined on the grid container. So, the simplest way is to control the padding from the grid container using CSS child combinator (`>`):
+
+```js
+function getOffset(val) {
+  const parse = parseFloat(val);
+  return `${parse}${String(val).replace(String(parse), '') || 'px'}`;
+}
+// ...The code is shorten for readability
+({
+  [`& > .${gridClasses.item}`]: {
+    // Grid item's padding is controlled by its container
+    paddingTop: getOffset(themeSpacing),
+  },
+});
+```
+
+However, in `Grid v2`, the javascript logic can be separated independently because CSS variables shift the communication between container and item to CSS.
+
+- **Grid container** takes care of the variables generation based on the `spacing` prop:
+  ```js
+  // The code is simplified for readability
+  {
+    '--Grid-rowSpacing': spacingProp,
+    '--Grid-columnSpacing': spacingProp,
+  }
+  ```
+- **Grid item** refers to the variables:
+  ```js
+  {
+    padding: `calc(var(--Grid-rowSpacing) / 2) calc(var(--Grid-columnSpacing) / 2)`,
+  }
+  ```
+
+This is a very good use case for applying CSS variables when it comes to components that has a parent-child relationship which also support media query customization.
+
+#### Remove unnecessary props
+
+In version 2, `Grid` is always an item which is similar to the flexbox item in CSS, so the `item` prop is not needed anymore.
+
+We also update the grid styles to work with ellipsis text by default, so the `zeroMinWidth` prop is removed.
```

Comment:
```suggestion
We've also updated the grid styles to work with ellipsis text by default, so the `zeroMinWidth` prop has been removed:
```

Reply:
What exactly is meant by "work with ellipsis text"? Does that mean it supports text that includes an ellipsis?

Reply:
My intention is to say that developers do not need to use `zeroMinWidth` together with `<Typography nowrap>` anymore.

https://mui.com/material-ui/react-grid/#white-space-nowrap

Reply:
Ah ok, that's a helpful example. I think I would describe it like this (if I'm understanding correctly):

```suggestion
The new `Grid` automatically handles the sizing of truncated text that's too long for its container, so we've also removed the `zeroMinWidth` prop since it's no longer needed:
```

Reply:
awesome!